clients/pkg/promtail/scrapeconfig/scrapeconfig.go

```diff
@@ -202,12 +211,19 @@ type SyslogTargetConfig struct {
 	// message should be pushed to Loki
 	UseRFC5424Message bool `yaml:"use_rfc5424_message"`
 
+	// Syslog format used at the target.
```

Comment:
I think we should clarify the valid options here, and that `rfc5424` is the default.

Reply:
we could implement the default in the parser or like this: https://github.com/grafana/loki/blob/efdae3df14c47d627eb99e91466e0451db6e16f6/clients/pkg/promtail/targets/syslog/syslogtarget.go#L233-L238

Reply:
It looks like we don't have a dedicated config parsing function on this path, so maybe in `NewSyslogTarget` would make the most sense to detect if `SyslogFormat` is empty string and if so set the default